p2p/discovery/udp.go

```diff
@@ -5,146 +5,312 @@
 package discovery
 
 import (
+	"container/list"
+	"github.com/seeleteam/go-seele/crypto"
 	"net"
+	"time"
 
 	"github.com/seeleteam/go-seele/common"
 	"github.com/seeleteam/go-seele/log"
 )
 
+const (
+	responseTimeout = 500 * time.Millisecond
```

Comment:
500ms seems not enough for some bad network conditions

Reply:
change to 2 seconds